modules/local/asteroid.nf

```diff
@@ -29,7 +29,11 @@ process ASTEROID {
     def args = task.ext.args ?: ''
     """
     # Run asteroid using the multithreaded mpi version
-    mpiexec -np ${task.cpus} --allow-run-as-root asteroid -i $treefile -m $asteroid_map -p asteroid
+    mpiexec -np ${task.cpus} --allow-run-as-root --use-hwthread-cpus \
```

Comment:
Do you need both `-np ${task.cpus}` and `--use-hwthread-cpus`?

Reply:
i believe so: -np specifies the number of simultaneous processes to run, and --use-hwthread-cpus specifies that threads are treated as cpus. When calling mpiexec (or mpirun) you must specify the number of processes being run.